pkg/runtime/pubsub/cloudevents.go

```diff
@@ -29,13 +29,15 @@ type CloudEvent struct {
 	DataContentType string
 	TraceID         string
 	TraceState      string
+	ActorType       string
```

Comment:
Do we want to rename it to "DaprActorType" and "DaprActorId" to avoid conflict with existing names applications might be using?

Reply:
We could rename them. Will require update in the components-contrib/pubsub/envelope.go file